src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -15,15 +15,62 @@ module.exports = class LevelPlane extends Array {
     }
   }
 
+  inBounds(position) {
```

Comment:
In the future we should probably update all `inBounds` calls to reference this method, and remove `GameController.prototype.inBounds`.